lusidtools/cocoon/cocoon_printer.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+import numpy as np
+
+
+def check_errors_and_success(target_dict: dict, target_name: str, required_keys: list):
```

Comment:
Rename this function as it is not just for checking errors and success as you can provide any keys! This actually just checks a dictionary has the provided keys.